modules/statics/src/networks.ts

```diff
@@ -438,6 +438,19 @@ class Goerli extends Testnet implements EthereumNetwork {
   forwarderImplementationAddress = '0x80d5c91e8cc21df69fc4d64f21dc2d83121c3999';
 }
 
+class Holesky extends Testnet implements EthereumNetwork {
+  name = 'Holesky';
+  family = CoinFamily.ETH;
+  explorerUrl = 'https://holesky.beaconcha.in/tx/';
+  accountExplorerUrl = 'https://holesky.beaconcha.in/address/';
+  blockExplorerUrl = 'https://holesky.beaconcha.in/block/';
+  // https://chainlist.org/chain/17000
```

Comment:
nit: can add this as Ref - https://github.com/eth-clients/holesky#metadata

Reply:
yeah, that looks better. I will do it along with the next PR